src/controllers/category.js

```
@@ -0,0 +1,23 @@
+const categoryService = require('~/services/category')
+const { DEFAULT_CATEGORY_LIMIT, DEFAULT_CATEGORY_SKIP } = require('~/consts/category')
+
+const createCategory = async (req, res) => {
+  const categoryData = req.body
+
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)